### ISSUE TITLE
loss & accuracy definition

### ISSUE THREAD

**Issue by skw1990** [uber/ludwig#182]
Hi, 

Great tool so far. May I know if there's any good explanation on loss & accuracy, for instance: binary classification problem? Also, is there a more detailed documentation, like parameter tuning or so? 

Thanks.

**Comment by w4nderlust** [comment]
The binary classification loss is regular cross entropy, you can find descriptions of it everywhere, just google it.
Regarding hyperparameter tuning, the documentation is really detailed already, it will not become a machine learning 101 book, that would be nice but would require too much time unfortunately and i prefer to spend it on features and bugs as there are already thousands of other resources that teach machine learning and deep learning concepts, including what parameters to change and what is their effect.
What I could actually do is coming up with a curated list of resources for beginners, would that be useful for you?

**Comment by skw1990** [comment]
Thanks.

Actually just want to get clarification of the loss and accuracy definition. Loss could refer to other metrics say log loss.i presume accuracy here refers to predicted class using probability threshold and it's 0.5 by default right? 

On parameter tuning, would like to know for Ludwig tool case, what could be the best practice. For example, I am not sure how to do grid search on parameters used or any other parameter searching technique. 

Curated list of resources could be helpful too.

**Comment by w4nderlust** [comment]
Losses have names, if you look up their names you'll find definitions online. In the binary case you have a really classic cross entropy. For predictions, yes you can set the threshold and everything abouve that is true, and below if false.
For parameter tuning i would go and do random search as a first step, or you can use the nice sk-opt package too.

**Comment by w4nderlust** [comment]
Closing this as it is more a discussion than an issue, if you want to contonue it, please open a topic on Gitq.
